lib_xua/src/dfu/dfu.xc

```diff
@@ -41,10 +41,16 @@ static unsigned int DFUResetTimeout = 100000000; // 1 second default
 static int DFU_flash_connected = 0;
 
 static unsigned int subPagesLeft = 0;
+static int flash_cmd_start_write_image_in_progress = 1;
 
 extern void DFUCustomFlashEnable();
 extern void DFUCustomFlashDisable();
 
+static unsigned int save_blk0_request_data[16];
+
+#define DO_BLOCKING_ERASE (0) // If enabled, block on block0 download request while flash_cmd_start_write_image() erases the entire flash.
```

Comment:
Is there any benefit to having this set to 1? Maybe we just delete this to make the code simpler? (also its not tested, I assume)

Reply:
It does work when set to 0. I've removed it to not have code untested in the regression tests hanging about.